### ISSUE TITLE
SEO Meta is being cached globally instead of for each site

### ISSUE THREAD

**Issue by riasvdv** [nystudio107/craft-seomatic#17]
When you have a multi-site install, the SEO meta gets cached without accounting for the site. 

So when I have a site with a sitename of "Site 1" and view a page on it, the site name stays the same in the title even when viewing a page on site 2.

**Comment by khalwat** [comment]
Hrm. Are we sure it's caching that's the problem? The meta container case does seem to account for the `$siteId`:

https://github.com/nystudio107/craft-seomatic/blob/v3/src/services/MetaContainers.php#L135

```
$cache->getOrSet(
                $this::CACHE_KEY.$uri.$siteId,
                function () use ($uri, $siteId) {```

**Comment by khalwat** [comment]
Fixed in 3.0.0-beta.5:

https://github.com/nystudio107/craft-seomatic/releases/tag/3.0.0-beta.5
